### PR TITLE
Profile Page Updated

### LINKED ISSUE #3: Setup the Profile Page
Make the Profile component generic. Accept the props `userId`. Upon loading the page, make a Fetch API call to fetch the relevant data about the user. This may include the following entities:
- Name
- Programme
- Department
- Roll No 
All these fields shall be non-editable.<br/>
The following fields are optional and can be edited by the user.
- Resume Link (drive link)
- Github profile link
- LinkedIn profile link<br/>
Create a dummy function which would be called upon clicking the `Save` button. This function shall update the user database.

### PR DESCRIPTION
**Pull request by Shubham63-hub**
Screenshot from 2021-06-06 19-00-10

closes #3

### CODE REVIEW

File: src/views/examples/Profile.js [hunk 1]
```diff
@@ -161,173 +108,179 @@ const Profile = () => {
...
                 </Form>
               </CardBody>
             </Card>
           </Col>
         </Row>
+        <div className="text-center mt-3">
+            <Button
+              color="primary"
+              href="#pablo"
+              onClick={(e) =>{ console.log("save-clicked"); e.preventDefault();}}
```
Comment (mridul911) [inline src/views/examples/Profile.js]: Define a function that calls when someone clicks this button and as of now that function should console current values of Resume Link, Github link, etc.

File: src/components/Headers/UserHeader.js [hunk 2]
```diff
@@ -20,17 +20,17 @@ import React from "react";
...
         style={{
           minHeight: "600px",
-          backgroundImage:
-            "url(" +
-            require("../../assets/img/theme/profile-cover.jpg").default +
-            ")",
+          // backgroundImage:
+          //   "url(" +
+          //   require("../../assets/img/theme/profile-cover.jpg").default +
+          //   ")",
```
Comment (mridul911) [inline src/components/Headers/UserHeader.js]: Delete these comments if not needed.

File: src/views/examples/Profile.js [hunk 3]
```diff
@@ -161,173 +108,179 @@ const Profile = () => {
...
+}
+
+Profile.defaultProps = {
+  userID: 1,
+  name: "Name",
+  programme: "B.tech/B.S.",
+  department: "dep",
+  rollNo : 190833,
+  // resumeLink : "asdohfao"
+}
```
Comment (mridul911) [inline src/views/examples/Profile.js]: Pass these values from props instead of initializing like this.

File: src/views/examples/Profile.js (hunk 1 above)
Comment (kartikcode) [reply src/views/examples/Profile.js]: Maybe create a dummy API call too in the function. The request mode would be `POST`.

**Comment by Shubham63-hub** [comment]
I did all the changes which I could do discussing with Mridul.

**Comment by kartikcode** [comment]
Great!

**Pull request merged**